src/dynamics/video_player/player/player.js

```diff
@@ -811,7 +811,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         if (width) styles.width = isNaN(width) ? width : parseFloat(width).toFixed(2) + "px";
                         containerStyles = floatingonly ? {} : Objs.extend({}, styles);
                         if (!gallerySidebar && showsidebargallery && layout === "desktop" && !fullscreened)
-                            containerStyles.aspectRatio = this.get("sidebaroptions.aspectratio") || 838 / 360;
+                            if (!this.get('outstream')) containerStyles.aspectRatio = this.get("sidebaroptions.aspectratio") || 838 / 360;
```

Comment:
Hi @Josewd, same here. 🙂

Reply:
Hi @ntshcalleia do we have case where w're changing outstream on the fly? Because on this way we can load additional event listener to the browser.

Reply:
@sambua in some cases the player parameters can be updated asynchronously.